src/Symfony/Component/HttpFoundation/Cookie.php

```diff
@@ -66,17 +68,17 @@ public static function fromString(string $cookie, bool $decode = false): static
             $data['expires'] = time() + (int) $data['max-age'];
         }
 
-        return new static($name, $value, $data['expires'], $data['path'], $data['domain'], $data['secure'], $data['httponly'], $data['raw'], $data['samesite']);
+        return new static($name, $value, $data['expires'], $data['path'], $data['domain'], $data['secure'], $data['httponly'], $data['raw'], $data['samesite'], $data['partitioned']);
     }
 
     /**
      * @see self::__construct
      *
      * @param self::SAMESITE_*|''|null $sameSite
      */
-    public static function create(string $name, string $value = null, int|string|\DateTimeInterface $expire = 0, ?string $path = '/', string $domain = null, bool $secure = null, bool $httpOnly = true, bool $raw = false, ?string $sameSite = self::SAMESITE_LAX): self
+    public static function create(string $name, string $value = null, int|string|\DateTimeInterface $expire = 0, ?string $path = '/', string $domain = null, bool $secure = null, bool $httpOnly = true, bool $raw = false, ?string $sameSite = self::SAMESITE_LAX, bool $partitioned = false): self
```

Comment:
Adding a new argument in a non-final method is a BC break (even when static as a child class could redefine the static method and PHP would enforce the signature compatibility).
This must be added as a forward-compability argument instead.

Reply:
That's right, completely missed that. It's now implemented as a forward-compability argument. I also changed the tests and suggest to keep the assertions by using object instantiation. 

What do you think about another PR to extend the `create` method with the `$partitioned` argument in the next major release?